Wrappers/Python/ccpi/optimisation/operators/GradientOperator.py

```diff
@@ -409,8 +380,10 @@ def adjoint(self, x, out=None):
             return_val = True
 
         ndout, out_p = Gradient_C.datacontainer_as_c_pointer(out)
+        
+        tmp = x/self.voxel_size_order
 
-        arg1 = [Gradient_C.datacontainer_as_c_pointer(x.get_item(i))[1] for i in range(self.gm_range.shape[0])]
+        arg1 = [Gradient_C.datacontainer_as_c_pointer(tmp.get_item(i))[1] for i in range(self.gm_range.shape[0])]
```

Comment:
I think there is still a risk that 'tmp' will go out of scope after this call and before the c code has run. I guess maybe have a dummy use of tmp after the c call might work to make sure python doesn't throw it away.